Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Ride.hs

```diff
@@ -264,9 +271,11 @@ type RideRouteAPI =
 data ActualRoute = ActualRoute
   { lat :: Double,
     lon :: Double,
-    timestamp :: UTCTime
+    timestamp :: UTCTime,
+    accuracy :: Double,
+    rideStatus :: Status
   }
-  deriving stock (Show, Generic)
+  deriving stock (Show, Generic, Read)
```

Comment:
I am pretty sure this is redundant here and for `DriverEdaKafka`.
It's not really affect the logic, but affect code clarity since redundant instances can create false idea of how this is used or supposed to be used. Also it increases compilation time.